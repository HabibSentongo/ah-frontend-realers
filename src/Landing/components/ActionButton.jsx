import React from 'react';
import { Row, Col } from 'react-bootstrap';
import CustomButton from '../../commons/containers/CustomButton';

export default function ActionButton() {
  return (
    <Row>
      <Col sm={12} className="call-for-action">
        <CustomButton btnlabel="See More Articles" link="/articles" />
      </Col>
    </Row>
  );
}
