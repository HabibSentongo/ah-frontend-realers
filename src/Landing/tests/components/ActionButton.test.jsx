/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ActionButton from '../../components/ActionButton';

describe('ActionButton Component', () => {
  it('should match the snapshot', () => {
    const component = shallow(<ActionButton />);
    expect(toJson(component)).toMatchSnapshot();
  });
});
