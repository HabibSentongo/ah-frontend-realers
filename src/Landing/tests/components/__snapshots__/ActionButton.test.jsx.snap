// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ActionButton Component should match the snapshot 1`] = `
<Bootstrap(Row)>
  <Col
    className="call-for-action"
    sm={12}
  >
    <CustomButton
      btnlabel="See More Articles"
      link="/article"
    />
  </Col>
</Bootstrap(Row)>
`;
